corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -511,6 +511,64 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
             </div>
             {# END Roles & Permissions permissions #}
 
+            <legend>
+              {% trans "Reports" %}
+            </legend>
+
+            <div class="form-group">
+              <label class="col-sm-4 control-label">
```

Comment:
I think this should be hidden if the user's subscription does not have access to the report builder feature

Reply:
How can we implement that check?

Reply:
It looks like this util function does so https://github.com/dimagi/commcare-hq/blob/a37dc36631dbff5f5c63a9d9b49b83a16ded1221/corehq/apps/userreports/util.py#L49

Reply:
Thanks for the pointer. I've added has_report_builder_access to the context and conditionalized this item in the dialog.